src/xml.c

```diff
@@ -202,7 +202,7 @@ xhtml_extract_from_string (const gchar *html, const gchar *defaultBase)
 	if (body != NULL)
 		result = xhtml_extract (body, 1, defaultBase);
 	else
-		result = xmlCharStrdup ("<div></div>");
+		result = g_strdup ("<div></div>");
```

Comment:
If var body is NULL var result is allocated by g_strdup() otherwise by xmlCharStrdup() but they are always deallocated by xmlFree() see:
https://github.com/lwindolf/liferea/blob/71beee94a212387917a47e21f338b20e35e7c3ce/src/fl_sources/json_api_mapper.c#L154
https://github.com/lwindolf/liferea/blob/71beee94a212387917a47e21f338b20e35e7c3ce/src/fl_sources/ttrss_source_feed.c#L87